Userland/Libraries/LibDSP/Transport.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, kleines Filmröllchen <malu.bertsch@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include "Music.h"
+#include <AK/Types.h>
+#include <LibCore/Object.h>
+
+namespace LibDSP {
+
+// The DAW-wide timekeeper and synchronizer
+class Transport final : public Core::Object {
+public:
+    Transport(u16 beats_per_minute, u8 beats_per_measure)
+        : m_beats_per_minute(beats_per_minute)
+        , m_beats_per_measure(beats_per_measure)
+    {
+    }
+
+    const u32& time() const { return m_time; }
+    u16 beats_per_minute() const { return m_beats_per_minute; }
+    double current_second() const { return m_time / sample_rate; }
+    double samples_per_measure() const { return (1.0 / m_beats_per_minute) * 60.0 * sample_rate; }
+    double current_measure() const { return m_time / samples_per_measure(); }
+
+private:
+    const u32 m_time { 0 };
+    const u16 m_beats_per_minute;
+    const u8 m_beats_per_measure;
```

Comment:
East `const`. Also, please initialize all primitives to avoid hard to find mistakes. I know you set them in the constructor, but what if someone uses the default constructor? If they are `0`, you will at least notice that something is wrong.